src/main/java/fi/okm/jod/yksilo/config/SecurityConfig.java

```diff
@@ -46,25 +46,25 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
             })
         .build();
   }
-}
 
-final class CacheControlHeadersWriter implements HeaderWriter {
-  /*
-   * https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#dont_cache
-   *
-   * stale-if-error=0, see:
-   * https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html#ExpirationDownloadDist
-   * If the origin is [now] unreachable and the minimum or maximum TTL value is greater than 0,
-   * CloudFront will serve the object that it got from the origin previously. To avoid this
-   * behavior, include the Cache-Control: stale-if-error=0 directive.
-   */
-  private static final String CACHE_CONTROL_HEADER_VALUE =
-      "private, no-cache, no-store, stale-if-error=0";
+  static class CacheControlHeadersWriter implements HeaderWriter {
```

Comment:
I would keep the final modifier since this class is not intended to be extended (it might also help in some runtime optimizations)